src/intern.rs

```diff
@@ -506,6 +518,112 @@ mod intern_tests {
             h.join().unwrap()
         }
     }
+
+    #[cfg(feature = "deepsize")]
+    #[test]
+    fn test_deep_size() {
+        let string1 = String::from("abcdefghijklmnopqrstuvwxyz");
+        let string2 = String::from("abcdefghijklmnopqrstuvwxyz");
+        let string3 = String::from("abcdefghijklmnopqrstuvwxyz");
```

Comment:
Might be a little more clear with clone
```suggestion
        let string2 = string1.clone();
        let string3 = string1.clone();
```